client/web/compose/src/components/PageBlocks/ChartBase.vue

```diff
@@ -217,3 +304,17 @@ export default {
   },
 }
 </script>
+
+<style lang="scss" scoped>
+.chart__livefilter-btn {
+  display: flex;
```

Comment:
The only thing that should be here is (or you can add it to the style itself):
right: 0.75rem
margin-top: 0.25rem
z-index: 1;

Other things can be in the template class (position-absolute, p-1 and so on)
border and margin-left are redundant!